doc/api/url.md

```diff
@@ -1317,7 +1364,8 @@ The `url.format()` method returns a formatted URL string derived from
 `urlObject`.
 
 ```js
-url.format({
+const {format} = require('url');
+format({
```

Comment:
`format` is not a very descriptive by itself (contrarily to `pathToFileURL`). That wouldn't be consistent but at least it demonstrates that both imports are valid 🙃
```suggestion
const url = require('url');
url.format({
```